streams/es_rakuten.m3u

```diff
@@ -95,41 +83,41 @@ https://rakuten-topfree-3-eu.rakuten.wurl.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="error",Rakuten TV Shows 1 (720p)
-https://rakuten-tvshows-1-eu.rakuten.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="error",Rakuten TV Shows 2 (720p)
-https://rakuten-tvshows-2-eu.rakuten.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="error",Rakuten TV Shows 3 (720p)
-https://rakuten-tvshows-3-eu.rakuten.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="error",Rakuten TV Shows 6 (720p)
-https://rakuten-tvshows-6-eu.rakuten.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="error",Rakuten TV Shows 7 (720p)
-https://rakuten-tvshows-7-eu.rakuten.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="error",Travel XP (720p)
-https://travelxp-travelxp-1-eu.rakuten.wurl.com/manifest/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Action Movies 1 (720p)
+https://rakuten-actionmovies-1-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Action Movies 2 (720p)
+https://rakuten-actionmovies-2-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Action Movies 3 (720p)
+https://rakuten-actionmovies-3-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Comedy Movies 1 (720p)
+https://rakuten-comedymovies-1-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Comedy Movies 2 (720p)
+https://rakuten-comedymovies-2-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Comedy Movies 3 (720p)
+https://rakuten-comedymovies-3-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Comedy Movies 6 (720p)
+https://rakuten-comedymovies-6-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Comedy Movies 7 (720p)
+https://rakuten-comedymovies-7-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Spotlight 1 (720p)
+https://rakuten-spotlight-1-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Spotlight 2 (720p)
+https://rakuten-spotlight-2-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Spotlight 3 (720p)
+https://rakuten-spotlight-3-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Spotlight 6 (720p)
+https://rakuten-spotlight-6-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten Spotlight 7 (720p)
+https://rakuten-spotlight-7-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 1 (720p)
+https://rakuten-tvshows-1-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 2 (720p)
+https://rakuten-tvshows-2-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 3 (720p)
+https://rakuten-tvshows-3-eu.rakuten.wurl.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Rakuten TV Shows 6 (720p)
+https://rakuten-tvshows-6-eu.rakuten.wurl.tv/playlist.m3u8
```

Comment:
Anything having -6 on the URL seems to be Italian. Maybe splitting the links should be better ?

Reply:
and the -5 are German.